File-Formats/PDF/Working-with-DigitalSignature.md

```diff
@@ -5418,33 +4792,28 @@ You can download a complete working sample from [GitHub](https://github.com/Sync
 
 ## Long Term Validation (LTV) information
 
-Added support for LTV validation and getting CRL and OCSP embedded details from the digital signature.
+Added support for LTV validation and getting CRL and OCSP embedded details from the digital signature using [LtvVerificationInfo](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureValidationResult.html#Syncfusion_Pdf_Security_PdfSignatureValidationResult_LtvVerificationInfo) property.
```

Comment:
Added support for LTV validation and getting CRL and OCSP embedded details from the digital signature using [LtvVerificationInfo](https://help.syncfusion.com/cr/file-formats/Syncfusion.Pdf.Security.PdfSignatureValidationResult.html#Syncfusion_Pdf_Security_PdfSignatureValidationResult_LtvVerificationInfo) class.

Reply:
Changed the content